Lib/test/_test_multiprocessing.py

```diff
@@ -269,6 +269,67 @@ def _test(cls, q, *args, **kwds):
             q.put(bytes(current.authkey))
             q.put(current.pid)
 
+    def test_parent_process_attributes(self):
+        if self.TYPE == "threads":
+            self.skipTest('test not appropriate for {}'.format(self.TYPE))
+        from multiprocessing.process import current_process
```

Comment:
Shouldn't this be at the top-level?

Reply:
I often see `import` statements inside tests, and I have to admit I do not know what rule is used to determine if an import should be top-level or not.

Reply:
Generally, it's better to put imports at the top level, except if the module might not be available, or if importing it may have undesirable side-effects.

Reply:
By the way `multiprocessing.current_process()` works, and perhaps `multiprocessing.parent_process()` should work, too.